typescript/sdk/src/gas/token-prices.ts

```diff
@@ -97,6 +97,10 @@ export class CoinGeckoTokenPriceGetter implements TokenPriceGetter {
     );
   }
 
+  static getSecretName(environment: string): string {
```

Comment:
this feels very infra specific that should probably be there instead of the sdk

Reply:
Good call out, I'll move this out